lib/procore/auth/stores/redis.rb

```diff
@@ -13,7 +13,7 @@ def save(token)
         end
 
         def fetch
-          return unless redis.exists(redis_key)
+          return unless redis.exists?(redis_key)
```

Comment:
You could bypass the issue and instead of hitting redis with an [exists](https://redis.io/commands/exists/) followed by a [get](https://redis.io/commands/get/) (which returns `nil` if the key is not present, just ask for it.

```
json_token = redis.get(redis_key) || return
token = JSON.parse(json_token)
...
```
or
```
json_token = redis.get(redis_key)
return if json_token.nil?

token = JSON.parse(json_token)
...
```
whichever feels cleaner to you